include/swift/Option/Options.td

```diff
@@ -368,6 +368,11 @@ def emit_objc_header_path : Separate<["-"], "emit-objc-header-path">,
          ArgumentIsPath]>,
   MetaVarName<"<path>">, HelpText<"Emit an Objective-C header file to <path>">;
 
+def emit_xctest_methods_path : Separate<["-"], "emit-xctest-methods-path">,
+Flags<[FrontendOption, NoInteractiveOption, DoesNotAffectIncrementalBuild,
+ArgumentIsPath]>,
+MetaVarName<"<path>">, HelpText<"Emit a file containing XCTest test methods to <path>">;
```

Comment:
This affects the incremental build in that the first time you pass it you need to generate the thing. We should probably have a separate flag for that at some point, but we don't yet.